packages/cli/test/generate-index.spec.ts

```diff
@@ -51,6 +51,51 @@ describe('build index', () => {
             ].join('\n')
         );
     });
+    it('should create index file importing all matched stylesheets in outDir (outputSources)', async () => {
+        const fs = createMemoryFs({
+            src: {
+                '/compA.st.css': `
+               .a{}
+            `,
+                '/a/b/comp-B.st.css': `
+               .b{}
+            `,
+            },
+        });
+
+        const stylable = new Stylable({
+            projectRoot: '/',
+            fileSystem: fs,
+            requireModule: () => ({}),
+        });
+
+        await build(
+            {
+                outDir: './dist',
+                srcDir: './src',
+                indexFile: 'index.st.css',
+                outputSources: true,
```

Comment:
we need test when `outputSources` is false and we emit only js files

Reply:
we do not emit `js` files by default (in this branch)